eve/main.yml

```diff
@@ -1798,6 +1799,27 @@ stages:
           stage_names:
             - single-node-install
 
+  single-node-install-rocky-8:
+    worker:
+      type: local
+    steps:
+      - SetProperty:
+          name: Set OS property to Rocky 8
+          property: os
+          value: "rocky-8"
+      - SetProperty:
+          name: Set offline property to false
+          property: offline
+          value: "false"
+      - SetProperty:
+          name: Set use_proxy property to false
+          property: use_proxy
+          value: "false"
```

Comment:
It does not work on rocky as well ?

Reply:
Currently investigating this, but it seems there is an issue with the proxy setup script.
If I can't manage to find the reason soon, we will keep it as is for the moment.

Reply:
Sure :+1: